exporters/aerospike/aerospike-config.yml.sample

```diff
@@ -0,0 +1,26 @@
+integrations:
+    - name: nri-aerospike
+      config:
+        # API URL of the aerospike service
+        aerospike_db_host: localhost
+        aerospike_db_port: 3000
+
+        # Port to expose scrape endpoint on, If this is not provided a random port will be used to launch the exporter
+        exporter_port: 9145
+
+        # Cluster name is used on the aerospike.prometheus.json.template
```

Comment:
This line could be confusing for the customer. I think it is better to tell them why setting that is important

Reply:
done 👍